dds/DCPS/transport/framework/TransportRegistry.cpp

```diff
@@ -108,6 +108,7 @@ TransportRegistry::load_transport_configuration(const OPENDDS_STRING& file_name,
        ++index) {
     if (ACE_OS::strcmp(sect_name.c_str(), TRANSPORT_SECTION_NAME) == 0 ||
         ACE_OS::strcmp(sect_name.c_str(), TRANSPORT_TEMPLATE_SECTION_NAME) == 0) {
+      const bool is_template = ACE_OS::strcmp(sect_name.c_str(), TRANSPORT_TEMPLATE_SECTION_NAME) == 0;
```

Comment:
The same `strcmp` call is performed twice. Can `is_template` be created before the `if` block?